react/javascript/package.json

```diff
@@ -11,7 +11,7 @@
   "author": "Aslak Hellesøy",
   "license": "MIT",
   "scripts": {
-    "prepare": "make .codegen",
+    "prepare": "make .codegen && npx -y cpy-cli '**/*.scss' '../dist/src' --cwd='src' --parents",
```

Comment:
I think we should add `cpy-cli` to `devDependencies` to avoid the prompt if we want to install it?